### DISCUSSION TITLE
Home Assistant RPi base station

### DISCUSSION THREAD

**Discussion by grayfrost90** [ESPresense/ESPresense#407] [Q&A]
I was thinking that I really could do with an ESPresense device in my office, but it occurred to me that my Home Assistant Raspberry Pi is also in there, so I wondered if it's possible to run something on that to integrate into ESPresense?

There's the BLE monitor in HACS, but I'm not sure how that could work with the presence detection in ESPresense, i.e. I assume I'd effectively end up with two entries for my devices - one from ESPresense and one from the BLE monitor?

anyone else tried this?

**Comment by TDX44** [comment]
I was also looking into this. I'm running Wyoming on a RP4 and espresense on an ESP32 and want to combine both on one RPi4. Is this possible?
